gsa/src/web/entity/tags.js

```diff
@@ -179,26 +179,28 @@ class EntityTagsTable extends React.Component {
                       <TableData>
                         {tag.comment}
                       </TableData>
-                      <TableData>
-                        <IconDivider>
-                          <Icon
-                            img="disable.svg"
-                            value={tag}
-                            title={_('Disable Tag')}
-                            onClick={onTagDisableClick}
-                          />
-                          <DeleteIcon
-                            value={tag}
-                            title={_('Remove Tag from {{type}}',
-                              {type: typeName(entityType)})}
-                            onClick={() => onTagRemoveClick(tag.id, entity)}
-                          />
-                          <EditIcon
-                            value={tag}
-                            title={_('Edit Tag')}
-                            onClick={this.handleEditTag}
-                          />
-                        </IconDivider>
+                      <TableData align="center">
+                        <span>
```

Comment:
Is the span really necessary? At least it shouldn't.

Reply:
Did you try setting the align at the IconDivider?

Reply:
Yes, I did, but it didn't work. What I did not test, though, is to set the IconDivider to grow. A growing divider plus the align="center" in its tag solves the issue.